### PR TITLE
[Initializers] graphql-let error when bootstrapping empty Nextjs app

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Description / Motivation
* Remove `graphql-let` command when create empty app 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by sc-addypathania — APPROVED** [review]
Looks good

File: packages/create-sitecore-jss/src/initializers/nextjs/index.ts [hunk 1]
```diff
@@ -50,6 +56,15 @@ export default class NextjsInitializer implements Initializer {
       addInitializers = addInitAnswer.addInitializers;
     }
 
+    if (!addInitializers.includes('nextjs-styleguide')) {
```
Comment (ambrauer) [inline packages/create-sitecore-jss/src/initializers/nextjs/index.ts:59]: Should we also check `args.templates` for 'nextjs-styleguide'? e.g. if they used `--templates nextjs,nextjs-styleguide`

File: packages/create-sitecore-jss/src/initializers/nextjs/index.ts [hunk 2]
```diff
@@ -50,6 +56,15 @@ export default class NextjsInitializer implements Initializer {
...
+      const pkgPath = path.resolve(`${answers.destination}${sep}package.json`);
+      const pkg = openPackageJson(pkgPath);
+
+      pkg.scripts.bootstrap = pkg.scripts.bootstrap.replace(' && graphql-let', '');
+
+      writeFileToPath(pkgPath, JSON.stringify(pkg, null, 2));
```
Comment (ambrauer) [inline packages/create-sitecore-jss/src/initializers/nextjs/index.ts:65]: We have `writePackageJson` helper also. Should we use that instead here?

File: packages/create-sitecore-jss/src/initializers/nextjs/index.ts (hunk 1 above)
Comment (illiakovalenko) [reply packages/create-sitecore-jss/src/initializers/nextjs/index.ts:59]: Good point, I missed it

File: packages/create-sitecore-jss/src/initializers/nextjs/index.ts (hunk 2 above)
Comment (illiakovalenko) [reply packages/create-sitecore-jss/src/initializers/nextjs/index.ts]: Sure, I will

**Review by ambrauer — APPROVED** [review]
Looks good!

**Pull request merged**